airflow/providers/amazon/aws/example_dags/example_s3_to_redshift.py

```diff
@@ -72,14 +71,12 @@ def _remove_sample_data_from_s3():
         postgres_conn_id='redshift_default',
         task_id='teardown__drop_table',
     )
-    teardown__task_remove_sample_data_from_s3 = PythonOperator(
-        python_callable=_remove_sample_data_from_s3, task_id='teardown__remove_sample_data_from_s3'
-    )
+
     (
-        [setup__task_add_sample_data_to_s3, setup__task_create_table]
+        [_add_sample_data_to_s3(), setup__task_create_table]
```

Comment:
Let's call this task outside the array, imho it will be easier to understand

Reply:
Sounds good; will update this and the DAG with a similar comment.